korge/src/jvmMain/kotlin/com/soywiz/korge/testing/KorgeScreenshotTester.kt

```diff
@@ -80,9 +80,10 @@ class KorgeScreenshotTester(
     suspend fun recordGolden(
         view: View,
         goldenName: String,
-        settingOverride: KorgeScreenshotValidationSettings = defaultValidationSettings
+        settingOverride: KorgeScreenshotValidationSettings = defaultValidationSettings,
+        includeBackgroundColor: Boolean = true
     ) {
-        val bitmap = view.renderToBitmap(views)
+        val bitmap = view.renderToBitmap(views, includeBackground = includeBackgroundColor)
```

Comment:
@soywiz - I believe there is a bug with `includeBackground`, as you can see from the image below. The red background was not actually included.

Reply:
Lets create an issue with that